docs/docsite/rst/porting_guides/porting_guide_2.9.rst

```diff
@@ -90,6 +90,10 @@ The following modules will be removed in Ansible 2.13. Please update update your
 
 * vyos_linkagg use :ref:`vyos_lag_interfaces <vyos_lag_interfaces_module>` instead.
 
+* purefa_facts use :ref:`purefa_info <purefa_info_module>` instead.
```

Comment:
It would be nice if you could move the new entries up a bit, so the resulting list is sorted alphabetically.